lang/python/python-requests/patches/Allow-urllib3-125x.patch

```diff
@@ -0,0 +1,32 @@
+Pull request: https://github.com/kennethreitz/requests/pull/5063
```

Comment:
nitpick: one good practice about patches is to number them;
in this case, it should be fine to leave them as-is here, since it will be remove when `requests` updates;

Reply:
Right, I forget to do this.

Reply:
Added number, also added note about their commit, which they have prepared in their branch.